release/tools/jaxrs.xml

```diff
@@ -211,11 +211,12 @@
         <!-- the com/sun/ts/lib/implementation/sun/jersey file -->
         <copy file="${ts.home}/src/com/sun/ts/lib/implementation/sun/jersey/TSWebConfiguration.java" 
               todir="${deliverable.bundle.dir}/src/com/sun/ts/lib/implementation/sun/jersey"/>
-        
+ <!--      
```

Comment:
we might need to update "Copyright (c) 2018, 2019" to "Copyright (c) 2018, 2020" in this file.

Reply:
updated copyright.